datayoga/blocks/map/block.py

```diff
@@ -12,10 +12,12 @@
 class Block(DyBlock):
     def init(self):
         logger.debug(f"Initializing {self.get_block_name()}")
-        self.expression = get_expression_class(self.properties["language"], json.dumps(self.properties["expression"]))
+        self.expression = expression.compile(self.properties["language"], json.dumps(self.properties["expression"]))
 
     def run(self, data: Any, context: Context = None) -> Any:
```

Comment:
data: Any should be replaced with data: List[Dict[str, Any]] and return type from Any to List[Dict[str, Any]].